earn/src/components/borrow/BorrowGraph.tsx

```diff
@@ -90,11 +109,31 @@ export default function BorrowGraph(props: BorrowGraphProps) {
   const { graphData } = props;
 
   const now = Date.now();
-  const filteredGraphData = graphData
-    .filter((item) => now - item.x.getTime() <= MILLIS_PER_WEEK)
-    .map((item) => {
-      return { ...item, x: item.x.toISOString() };
-    });
+
```

Comment:
We should check that there is > 1 (since we already know it isn't empty); otherwise, we will run into an error.